src/components/Main.tsx

```diff
@@ -80,14 +86,18 @@ export function Main({
       <div
         role="dialog"
         aria-modal="true"
-        aria-labelledby="consent-dialog-title"
+        aria-label={formatMessage(messages.modalAriaLabel)}
         aria-live="polite"
         className="modal-container"
         id="consentManagerMainDialog"
         ref={dialogRef}
       >
         <div role="document" className="modal-container-inner" tabIndex={0}>
-          <div role="document" className="inner-container">
+          <div
+            role="main"
```

Comment:
I don't think this role is appropriate? I think you're [only supposed to have one `main` per page](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/main_role#specifications:~:text=There%20should%20only%20be%20one%20main%20landmark%20role%20per%20document.) and there should be already one in the main web page content if the customer has built their site with attention to accessibility. That said, I'm not sure if the way we are using shadow doms affects this recommendation. I also don't know if when it says "per document" if it is using that term generically or if it means the `document` role.

Reply:
it definitely means the document role! and this dialog of ours is its own document (formatted like the one shown [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/document_role#description))